### PR TITLE
Add date and time picker to `<DateFilter />`

### PR DESCRIPTION
**Pull request by balanza**
# Description

Add a DateTime for selecting a specific date and time as a filter value.

### CODE REVIEW

File: assets/js/common/DateFilter/DateFilter.jsx [hunk 1]
```diff
@@ -48,12 +55,58 @@ const parseInputOptions = (options) =>
...
   if (typeof selectedId === 'string') {
     return options.find((option) => option[0] === selectedId);
   }
   return undefined;
 };
 
+function Tick() {
+  return (
+    <span className="absolute inset-y-0 right-0 flex items-center pr-4">
+      <svg
```
Comment (nelsonkopliku) [inline assets/js/common/DateFilter/DateFilter.jsx:70]: **suggestion:** we might use `EOS_CHECK` icon from eos icon-set.

File: assets/js/common/Input/Input.stories.jsx [hunk 2]
```diff
@@ -160,6 +160,12 @@ export const Date = {
   },
 };
 
+export const DateTime = {
+  args: {
+    type: 'datetime-local',
```
Comment (nelsonkopliku) [inline assets/js/common/Input/Input.stories.jsx:165]: **praise:** well done in leveraging browser's calendar.

**Review by nelsonkopliku — COMMENTED** [review]
Cool!

**question:** I was looking for 24h formatting, but I understood it being a bit of a pain. Same conclusion from you, I guess :sweat_smile:

File: assets/js/common/DateFilter/DateFilter.jsx (hunk 1 above)
Comment (balanza) [reply assets/js/common/DateFilter/DateFilter.jsx:70]: Good point. I'd rather do it in another PR, as we might also refactor the `Filter` component. I prepared it already: https://github.com/trento-project/web/pull/2870

**Review by nelsonkopliku — APPROVED** [review]

**Pull request merged**